vector/src/main/java/im/vector/app/features/home/room/threads/list/views/ThreadListFragment.kt

```diff
@@ -90,6 +91,12 @@ class ThreadListFragment @Inject constructor(
         renderToolbar()
     }
 
+    private fun initTextConstants() {
+        views.threadListEmptyNoticeTextView.text = String.format(
+                resources.getString(R.string.thread_list_empty_notice),
+                resources.getString(R.string.reply_in_thread))
```

Comment:
Why not using `getString(R.string.thread_list_empty_notice, getString(R.string.reply_in_thread))` ?